index.html

```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card2">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card3">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card4">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card5">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card6">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+    </section>
+
+    <section class="about_me">
+      <section class="description">
+        <h1>About me</h1>
```

Comment:
- [ ] To help improve your page SEO, it is best practice to have only one h1 tag on an HTML page. Kindly change all the h1 tags in each project card to any other hx, could be h2.